auth/api/logging.go

```diff
@@ -28,314 +27,422 @@ func LoggingMiddleware(svc auth.Service, logger *slog.Logger) auth.Service {
 
 func (lm *loggingMiddleware) ListObjects(ctx context.Context, pr auth.PolicyReq, nextPageToken string, limit int32) (p auth.PolicyPage, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method list_objects took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.String("duration", time.Since(begin).String()),
+			slog.Any("limit", limit),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.String("error", err.Error()))
```

Comment:
```suggestion
			args = append(args, slog.Any("error", err))
```

Do this for the rest. It returns a better error format